src/Cms/FileActions.php

```diff
@@ -191,7 +194,9 @@ public static function create(array $props)
 
 		// make sure that a UUID gets generated and
 		// added to content right away
-		$content['uuid'] = Uuid::generate();
+		if (Uuids::enabled() === true) {
+			$content['uuid'] ??= Uuid::generate();
+		}
```

Comment:
@distantnative Was there a particular reason why you used `??=` in `PageActions`, but `=` here? I think Ahmet's change to always use `??=` makes sense, just want to be sure.

Reply:
No, I think @afbora is right